frida_tools/application.py

```diff
@@ -132,6 +132,7 @@ def store_target(option, opt_str, target_value, parser, target_type, *args, **kw
         self._exit_status = None
         self._console_state = ConsoleState.EMPTY
         self._have_terminal = sys.stdin.isatty() and sys.stdout.isatty() and not os.environ.get("TERM", '') == "dumb"
+        self._plain_terminal = os.environ.get("TERM", '').lower() == "none"
```

Comment:
Default value should be `""`, not `''`. (The convention used is double-quoted strings for strings, and single-quoted for enum values. In hindsight kinda weird, but better stick to it for consistency.)